test/e2e-common.sh

```diff
@@ -52,18 +52,37 @@ function install_pipeline_crd() {
 
 function install_triggers_crd() {
   echo ">> Deploying Tekton Triggers"
-  ko apply -f config/ || fail_test "Tekton Triggers installation failed"
+  rel=$(mktemp)
+  release=$(mktemp)
+  ko resolve -f config/ > "${rel}" || fail_test "Tekton Triggers build failed"
+
+  if [ "${SKIP_SECURITY_CTX}" == "true" ]; then
+      yq 'del(.spec.template.spec.containers[]?.securityContext.runAsUser, .spec.template.spec.containers[]?.securityContext.runAsGroup)' "${rel}" > "${release}"
+  else
+      cat "${rel}" > "${release}"
+  fi
+
+  kubectl apply -f "${release}" || fail_test "Tekton Triggers installation failed"
 
   # Wait for the Interceptors CRD to be available before adding the core-interceptors
   kubectl wait --for=condition=Established --timeout=30s crds/clusterinterceptors.triggers.tekton.dev
-  ko apply -f config/interceptors || fail_test "Core interceptors installation failed"
+  ko resolve -f config/interceptors > "${rel}" || fail_test "Core interceptors build failed"
+
+  if [ "${SKIP_SECURITY_CTX}" == "true" ]; then
```

Comment:
I think we are setting SKIP_SECURITY_CTX as false do we still need this change??

Reply:
I didn't get this. We need this when SKIP_SECURITY_CTX is true.